terraform-dev/govgraphsearch.tf

```diff
@@ -189,6 +189,14 @@ resource "google_cloud_run_service" "govgraphsearch" {
       service_account_name = google_service_account.govgraphsearch.email
       containers {
         image = "europe-west2-docker.pkg.dev/${var.project_id}/${google_artifact_registry_repository.cloud_run_source_deploy.repository_id}/govuk-knowledge-graph-search:latest"
+        env {
+          name  = "REDIS_HOST"
+          value = google_redis_instance.session_store.host
```

Comment:
Because `google_redis_instance.session_store` now uses `count`, it is now an array with 0 or 1 elements (zero-indexed).  It can be referenced in a fail-safe way with:

```tf
value = try(google_redis_instance.session_store[0].host, "")
```

Similarly `google_redis_instance.session_store.port` below.

Reply:
Good to know, thanks!